editor/import/scene_import_settings.cpp

```diff
@@ -246,6 +246,17 @@ void SceneImportSettings::_fill_scene(Node *p_node, TreeItem *p_parent_item) {
 		p_node->set_meta("import_id", import_id);
 	}
 
+	Skeleton3D *skeleton = Object::cast_to<Skeleton3D>(p_node);
+	if (skeleton) {
+		Node3D *skeleton_parent = skeleton->get_parent_node_3d();
+		if (skeleton_parent) {
+			if (skeleton_parent->get_transform().basis.get_scale() != Basis().get_scale()) {
```

Comment:
I wonder why you're not calculating global scale here.  This is get parent's local scale which is still not global scale.

Reply:
In all of the Mixamo models I tried, the parent's global scale was (1, 1, 1).

Reply:
> In your other pr you have a for loop to calculate global scale.
That was for calculating the cumulative scale to set the LOD distance, but it wasn't necessary.